docs/3.0-migration-guide.md

```diff
@@ -13,7 +13,25 @@ The changes are detailed below, and include:
 - Removal of many insecure or obsolete features
 - Tidying up of configuration options (including removing some less useful options).
 - Changing function signatures, e.g. adding return codes, adding extra parameters, or making some arguments const.
-- Removal of functions previously marked as deprecated.
+- Removal of functions, macros, and types previously marked as deprecated.
+
+
+## Referring to Mbed TLS 2.x documentation
+
+For features previously marked as deprecated, Mbed TLS 2.x documentation may
+explain how to upgrade, and should be referred to when migrating code. Where a
+migration path is not provided in prior documentation, changes made and the
+upgrade steps required will be explained in this guide.
+
+To generate the documentation, checkout the `mbedtls-2.28` branch and follow
```

Comment:
So the section title and first paragraph talk about "2.x" documentation, but this is specifically for the 2.28 LTS branch.

Should we be talking about "Mbed TLS 2.28" rather than "2.x"?

Should someone check out the branch for the version of Mbed TLS 2.x they are upgrading from?

Or: is it because 2.28 is the last release of 2.x that it's the best place to look for this information?

If so, why do we refer to "2.x" everywhere instead of "2.28"?

How about

```suggestion
It's best to use the latest version of Mbed TLS 2.x for this purpose, which is the 2.28 LTS release.
To generate the documentation, checkout the `mbedtls-2.28` branch and follow
```

What do you think?

Reply:
I think the change is good to help to clarify why the 2.28 branch is used here, I'm unsure if using 2.28 instead of 2.x everywhere would further improve the clarity of this section or not.